linux/meta/sensu.yml

```diff
@@ -27,7 +27,7 @@ check:
     - {{ system.name|replace('.', '-') }}-{{ system.domain|replace('.', '-') }}
 {%- endif %}
   local_linux_storage_disk_usage:
-    command: "PATH=$PATH:/usr/lib64/nagios/plugins:/usr/lib/nagios/plugins check_disk -w 15% -c 5% -p / -p /var -p /usr -p /tmp -p /var/log"
+    command: "PATH=$PATH:/usr/lib64/nagios/plugins:/usr/lib/nagios/plugins check_disk -w" {{ monitoring.disk.get('warn', '15%') }} " -c " {{ monitoring.disk.get('critical', '5%') }}  " -p / -p /var -p /usr -p /tmp -p /var/log"
```

Comment:
No, it will not work at all:
1. monitoring is not defined (seems you didn't test this change), for what you are trying to achieve, this should be `pillar.monitoring` or `system.monitoring`, depends on agreed metadata structure for overriding defaults - @cznewt needs to give feedback here
2. ok, if `monitoring` is defined, you need to check for child structure as well, so it should be `monitoring.get('disk', {}).get('warn', '15%')`